docs/changes/0.14.0.rst

```diff
@@ -19,35 +19,42 @@ _________________
 * Remove deprecated ``zmq_helpers.py`` with ``Publisher`` and ``UnboundedPublisher`` (#1983)
 * Remove deprecated ``_save_val`` method of ``_BaseParameter`` (#1980)
 * Remove deprecated ``qcodes.dataset.sqlite_base`` and ``qcodes.dataset.database`` modules (#1981)
-* The ``name`` and the ``short_name`` attributes of the _BaseParameter have become read-only. It is not possible to change the name or short_name of the parameter after instantiation. (#1989)
-* ..
+* The ``name`` and the ``short_name`` attributes of the _BaseParameter have become read-only. It is not 
+  possible to change the name or short_name of the parameter after instantiation. (#1989)
+
 
 Improved drivers:
 _________________
 
-* ..
+* Improvements to ``KeysightB1520A`` to fix the following two issues, ``is_enabled` function 
+  was giving an error when called before enabling channels. This is fixed by checking if ``activated_channels`` 
+  is not empty string. Also adding ``get_cmd`` to ``voltage_dc``, ``voltage_ac``, ``frequency``. The above 
+  two fixes allows user to use the step attribute of the parameter to slowly raise/decrease a parameter. 
+  Before these fixes the user was getting a warning that the parameter is jumping to designated value. This 
+  because the starting value of the parameter was not found and there was no get_cmd available. (#1998)
```

Comment:
```suggestion
* Improvements to ``KeysightB1520A`` to fix the following two issues. First, ``is_enabled`` function outputs `False` instead of throwing an error if channels are not enabled. Second, the values of ``voltage_dc``, ``voltage_ac``, and ``frequency`` parameters can be raised/decreased at a specified rate by defining the step attribute of these parameters. (#1998)
```